successfactors-connector/processes/connector/PayGroup.p.json

```diff
@@ -0,0 +1,97 @@
+{
+  "format" : "10.0.0",
+  "id" : "18E82BA8F564A810",
+  "kind" : "CALLABLE_SUB",
+  "config" : {
+    "data" : "connector.PayGroupData"
+  },
+  "elements" : [ {
+      "id" : "f1",
+      "type" : "CallSubEnd",
+      "visual" : {
+        "at" : { "x" : 480, "y" : 88 }
+      }
+    }, {
+      "id" : "f3",
+      "type" : "CallSubStart",
+      "name" : "getPayGroups",
+      "config" : {
+        "callSignature" : "getPayGroups",
+        "input" : {
+          "params" : [
+            { "name" : "filter", "type" : "String" },
+            { "name" : "count", "type" : "Boolean" },
+            { "name" : "top", "type" : "Integer" },
+            { "name" : "skip", "type" : "Integer" },
+            { "name" : "orderby", "type" : "List<String>" },
+            { "name" : "select", "type" : "List<String>" },
+            { "name" : "expand", "type" : "List<String>" }
+          ],
+          "map" : {
+            "out.param.count" : "param.#count",
+            "out.param.expand" : "param.#expand",
+            "out.param.filter" : "param.#filter",
+            "out.param.orderBy" : "param.#orderby",
+            "out.param.select" : "param.#select",
+            "out.param.skip" : "param.#skip",
+            "out.param.top" : "param.#top"
+          }
+        },
+        "result" : {
+          "params" : [
+            { "name" : "payGroups", "type" : "List<com.axonivy.connector.successfactors.connector.rest.SFODataFOPayGroup>" }
+          ],
+          "map" : {
+            "result.payGroups" : "in.payGroups"
```

Comment:
Miss BpmError response.
Change `in.payGroups` to `in.#payGroups`
Ref to PayGrade
![image](https://github.com/axonivy-market/successfactors-connector/assets/84316773/ef0b3378-25e6-4731-b248-27c1318996b3)